packages/repository/src/__tests__/fixtures/mixins/baseEntity.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
```

Comment:
The file name uses camel case, which is not consistent with our code base. I guess you rename it to `base-entity.ts`.